File: scripts/check_rule22.py
import json

with open('bot/Rules_5.rules', 'r', encoding='utf-8') as f:
    rules = json.load(f)

# מצא את Rule 22
rule22 = next((r for r in rules if '22##' in r.get('Rule', '')), None)

if rule22:
    print("=== Rule 22 ===")
    print("Rule:", rule22['Rule'])
    print("\nResponse preview (first 600 chars):")
    print(rule22['response'][:600])
    print("\n...")
    
    # בדוק אם יש "בחר מודול"
    if 'בחר מודול' in rule22['response']:
        print("\n⚠ Still contains 'בחר מודול' - needs update")
    else:
        print("\n✓ Description updated correctly")
        
    if 'נקסוס בלבד' in rule22['response']:
        print("✓ Contains 'נקסוס בלבד'")
    else:
        print("⚠ Missing 'נקסוס בלבד'")
else:
    print("Rule 22 not found!")

File: scripts/check-rule22.js
const fs = require('fs');
const rules = JSON.parse(fs.readFileSync('bot/Rules_5.rules', 'utf8'));

// מצא את Rule 22
const rule22 = rules.find(r => r.Rule && r.Rule.includes('22##'));
if (rule22) {
  console.log('Rule 22 found:');
  console.log('Rule field:', rule22.Rule);
  console.log('\nResponse (first 500 chars):');
  console.log(rule22.response.substring(0, 500));
} else {
  console.log('Rule 22 not found');
}

// בדוק גם את התפריט הראשי (Rule 1)
const rule1 = rules.find(r => r.Rule && r.Rule.includes('1##התחל'));
if (rule1) {
  console.log('\n\n=== Rule 1 - Main Menu ===');
  const descMatch = rule1.response.match(/description:`([^`]+)`/g);
  if (descMatch) {
    descMatch.forEach(d => console.log(d));
  }
}
